library/oneview_login_detail_facts.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+###
+# Copyright (2016-2017) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: oneview_login_detail_facts
+short_description: Retrieve the facts about login details
+description:
+    - Retrieve the facts about login details from oneview.
+version_added: "2.5"
+requirements:
+    - hpOneView >= 4.3.0
+author:
+    - Madhav Bharadwaj(@madhav-bharadwaj)
```

Comment:
Since you're listing only one author, you should add this in the same line as `author:`
```
author: "Madhav Bharadwaj(@madhav-bharadwaj)"
```